Source/WebCore/html/parser/HTMLTreeBuilder.cpp

```diff
@@ -1414,9 +1464,42 @@ void HTMLTreeBuilder::processAnyOtherEndTagForInBody(AtomHTMLToken&& token)
     }
 }
 
+void HTMLTreeBuilder::processAnyOtherEndTagForUnknownElementInBody(AtomHTMLToken&& token)
```

Comment:
Can we use template & some lambda to share the code between the two?

Reply:
Yes we can.